floris/core/wake.py

```diff
@@ -53,6 +54,8 @@
         "jensen": JensenVelocityDeficit,
         "turbopark": TurbOParkVelocityDeficit,
         "empirical_gauss": EmpiricalGaussVelocityDeficit,
+        "turboparkgauss": TurboparkgaussVelocityDeficit,
+        # "doublegauss": DoublegaussVelocityDeficit,
```

Comment:
Same here - I haven't looked through all of the PR so maybe the `DoublegaussVelocityDeficit` is still coming?